presto-hive-hadoop2/pom.xml

```diff
@@ -186,15 +186,31 @@
             </build>
         </profile>
         <profile>
-            <id>test-hive-hadoop2-abfs</id>
+            <id>test-hive-hadoop2-abfs-access-key</id>
             <build>
                 <plugins>
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
                             <includes>
-                                <include>**/TestHiveFileSystemAbfs.java</include>
+                                <include>**/TestHiveFileSystemAbfsAccessKey.java</include>
+                            </includes>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>test-hive-hadoop2-abfs-oauth</id>
```

Comment:
Perhaps we could just keep a `test-hive-hadoop2-abfs1` profile to run both tests.

Reply:
We use the profiles to make the CI scripts run, so we would also need to merge those, which would be a bit trickier.

If we gave them completely disjoint test configuration properties, it wouldn't be too hard, but it might not look nice. On the other hand, it should let us just start the container once for both tests (we could also merge the WASB tests with them, as that is also ADLS Gen2).

Alternatively, we could remove the non-default Maven profiles for _all_ of the Hive storage tests except and just run the individual test classes.

Reply:
How much time does it take if we have those separate. If it does not make whole build longer I would not fight for that. We can possibly do that as a followup.

Reply:
> On the other hand, it should let us just start the container once for both tests (we could also merge the WASB tests with them, as that is also ADLS Gen2).

That's where I was going with this initially, but let's scrape that for now and stay focused.